controllers/drplacementcontrol.go

```diff
@@ -33,7 +33,7 @@ const (
 )
 
 var (
-	WaitForAppResouceRestoreToComplete error = errorswrapper.New("Waiting for App resources to be restored...")
+	WaitForAppResourceRestoreToComplete error = errorswrapper.New("Waiting for App resources to be restored...")
```

Comment:
If we touch this, maybe match the error message?

    WaitingForAppResourcesRestoreToComplete error = errorswrapper.New("Waiting for App resources to be restored...")

But it is very hard to work with such long names using camelCase style. We really much shorter names, how about:

    WaitingForRestore error = errorswrapper.New("Waiting for App resources to be restored...")